core/script_debugger_remote.cpp

```diff
@@ -899,6 +863,41 @@ void ScriptDebuggerRemote::send_message(const String &p_message, const Array &p_
 	mutex->unlock();
 }
 
+void ScriptDebuggerRemote::send_error(const String &p_func, const String &p_file, int p_line, const String &p_err, const String &p_descr, ErrorHandlerType p_type, const Vector<ScriptLanguage::StackInfo> &p_stack_info) {
```

Comment:
@reduz As I mentioned on IRC, I needed to add this function in order to display the exception stack info.